packages/cli/src/databases/entities/test-definition.ee.ts

```diff
@@ -27,6 +32,9 @@ export class TestDefinition extends WithTimestampsAndStringId {
 	@Column('text')
 	description: string;
 
+	@Column(jsonColumnType, { default: '[]' })
+	pinnedNodes: PinnedNodeItem[];
```

Comment:
Why are we duplicating the pinned nodes when they are already stored in the test workflow? I'm worried that it's too easy for them to get out of sync

Reply:
There is a naming confusion here, we use "pinned nodes" in workflow evaluation context for nodes which data should be mocked during the execution of workflow under test (it uses pin data feature of the workflow execution engine internally though).
We decided to change the naming to `mockedNodes` both in the code and in user-facing UI